### PR TITLE
NET-187: Event stream for storage node assignments

### PR DESCRIPTION
**Pull request by teogeb**
Listens a storage node assignments from `${EE_address}/storage-node-assignments` stream.

BREAKING CHANGE: There is a new `streamrAddress` setting in the config. That value is needed for storage nodes, other nodes can set that value to `null`. The `configs` directory contains valid defaults for the test environment.

### CODE REVIEW

File: src/storage/StorageConfig.js [hunk 1]
```diff
@@ -47,41 +55,77 @@ module.exports = class StorageConfig {
...
-                        streamKeys.push(getKeyFromStream(stream.id, i))
-                    }
+                    streamKeys = new Set([...streamKeys, ...getKeysFromStream(stream.id, stream.partitions)])
                 })
                 this._setStreams(streamKeys)
                 return undefined
             })
     }
 
+    startAssignmentEventListener(streamrAddress, networkNode) {
```
Comment (harbu) [inline src/storage/StorageConfig.js:113]: `startAssignmentStreamListener` or smth for clarity?

File: test/integration/broker.test.js [hunk 2]
```diff
@@ -36,27 +37,27 @@ describe('broker: end-to-end', () => {
...
             networkPort: networkPort1,
             trackerPort,
             httpPort: httpPort1,
             wsPort: wsPort1,
             enableCassandra: true
         })
-        broker2 = await startBroker({
-            name: 'broker2',
-            privateKey: broker2Key,
+        storageNode2 = await startBroker({
```
Comment (harbu) [inline test/integration/broker.test.js:49]: Does this test pass if we make 2/3 of the nodes non-storage? I think that has been the original intention...

File: test/unit/storage/StorageConfig.test.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-underscore-dangle, object-curly-newline, object-curly-spacing */
```
Comment (harbu) [inline test/unit/storage/StorageConfig.test.js:1]: These two `object-curly-newline, object-curly-spacing` lead to inconsistence style in the project...

**Review by harbu — CHANGES_REQUESTED** [review]

File: test/integration/broker.test.js (hunk 2 above)
Comment (teogeb) [reply test/integration/broker.test.js:49]: @harbu implements this refactoring in a separate PR

File: test/unit/storage/StorageConfig.test.js (hunk 3 above)
Comment (teogeb) [reply test/unit/storage/StorageConfig.test.js]: Fixed

**Review by harbu — APPROVED** [review]

**Pull request merged**